app/views/home/index.html.erb

```diff
@@ -1 +1 @@
-<%= render_react_component(component: "App", props: { user: @user }) %>
+<%= render_react_component(component: "App", props: { initial_user: @user }) %>
```

Comment:
why initial_user instead of user?

Reply:
To not conflict with the component internal `user`. This is the user that comes from the backend, and inside the component you also a have a user state, so you can logout by setting that to null.